src/adagrams.js

```diff
@@ -1,8 +1,118 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    // Declare letter distribution pool in an array
+    const pool = {
+      A: 9, B: 2, C: 2, D: 4, E: 12, F: 2, G: 3, H: 2,
+      I: 9, J: 1, K: 1, L: 4, M: 2, N: 6, O: 8, P: 2, Q: 1,
+      R: 6, S: 4, T: 6, U: 4, V: 2, W: 2, X: 1, Y: 2, Z: 1
+    };
+
+    // Create array for all possible letters
+    const fullDeck = []
+
+    for (let obj in pool) {
+      for (let i = 0; i < pool[obj]; i++) {
+        fullDeck.push(obj);
+      }
+    }
+
+    // Populate player's hand 
+    const lettersInHand = [];
+
+    // Create Sattolo Cycle implementation for shuffle
+    function sattoloShuffle(fullDeck) {
+      for (let f = fullDeck.length - 1; f >= 0; f--) {
+      const b = Math.floor(Math.random() * f)
```

Comment:
mind your indentation, starting here! :)

`const b =` ... should be indented one more, since you're in a for loop.